lib/types/init.lua

```diff
@@ -1,5 +1,7 @@
 local names = {
 	"Color3",
+	"UDim2",
```

Comment:
`UDim` is a fully constructable type, and is used in some of the layout objects; it should be publicly exposed.